base/java-tools/src/com/netscape/cmstools/CMCResponse.java

```diff
@@ -370,6 +370,14 @@ public static void main(String args[]) throws Exception {
             System.exit(1);
         }
 
+        //Intialize the crypto manager, just in case we need to use the JSS Provider to parse 
+        //algorithm parameters. All we have to do is initialize the manager and be done.
+
+        if (dbdir == null)
+            dbdir = ".";
```

Comment:
@jmagne -- as an alternative to `.`, which may or may not be a NSS DB, there is `/etc/pki/nssdb`, which is guaranteed to be a NSS DB because it is installed by NSS itself. It won't have any keys or anything, but initialization won't fail -- like it does if `.` isn't a NSS DB. Up to you though.